patroni/dcs/raft.py

```diff
@@ -223,50 +228,50 @@ def delete(self, key, recursive=False, **kwargs):
             return False
 
     @staticmethod
-    def __values_match(old, new):
+    def __values_match(old: Dict[str, Any], new: Dict[str, Any]) -> bool:
         return all(old.get(n) == new.get(n) for n in ('created', 'updated', 'expire', 'value'))
 
     @replicated
-    def _expire(self, key, value, callback=None):
+    def _expire(self, key: str, value: Dict[str, Any], callback: Optional[Callable[..., Any]] = None) -> None:
```

Comment:
```suggestion
    def _expire(self, key: str, value: Dict[str, Any]) -> None:
```

It seems we can remove `callback` parameter.

Reply:
No, we can't there is some magic in the `@replicated` decorator.